go.mod

```diff
@@ -7,26 +7,22 @@ require (
 	cloud.google.com/go v0.45.1
 	github.com/alecthomas/template v0.0.0-20190718012654-fb15b899a751 // indirect
 	github.com/alecthomas/units v0.0.0-20190717042225-c3de453c63f4 // indirect
-	github.com/aws/aws-sdk-go v1.28.13
+	github.com/aws/aws-sdk-go v1.29.10
```

Comment:
Unless a new dep is being added or it strictly needs to be upgraded, let's put the go.{mod,sum} changes in a separate PR. It reduces the (diff) size of the PR and facilitates reviewing.
Also, when running `go mod tidy`, make the branch is up to date with master to avoid removing things that are needed.
Also, given that some dependencies are not included with the default flags, it may make sense to have an explicit Makefile target to tidy up the modules files.